scoringrules/_energy.py

```diff
@@ -227,14 +227,14 @@ def vrenergy_score(
         forecasts, observations, m_axis, v_axis, backend=backend
     )
 
-    fcts_weights = B.apply_along_axis(w_func, forecasts, -1)
+    fct_weights = B.apply_along_axis(w_func, forecasts, -1)
     obs_weights = B.apply_along_axis(w_func, observations, -1)
 
     if backend == "numba":
         return energy._vrenergy_score_gufunc(
-            forecasts, observations, fcts_weights, obs_weights
+            forecasts, observations, fct_weights, obs_weights
         )
 
     return energy.vrnrg(
-        forecasts, observations, fcts_weights, obs_weights, backend=backend
+        forecasts, observations, fct_weights, obs_weights, backend=backend
```

Comment:
forecasts is still the first argument here, I need to check these energy functions